### PR TITLE
Added order data to OrderController index

### PR DESCRIPTION
**Pull request by ghermans**
This allows developers to use the data without using the default datagrid.

### Example
exampleKnipsel

### CODE REVIEW

**Comment by ghermans** [comment]
@jitendra-webkul can you please verify this PR

File: packages/Webkul/Shop/src/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -50,7 +50,13 @@ public function __construct(
     */
     public function index()
     {
-        return view($this->_config['view']);
+        $order = $this->orderRepository->findOneWhere(['customer_id' => auth()->guard('customer')->user()->id]);
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/OrderController.php:53]: This will always return first customer order. I don't see any purpose of this.

Comment (ghermans) [reply packages/Webkul/Shop/src/Http/Controllers/OrderController.php:53]: You are right, i only tested it with one order, i will provide a fix for this

**Comment by ghermans** [comment]
@jitendra-webkul this should be fixed now.

**Comment by jitendra-webkul** [comment]
@ghermans, We can fetch orders in blade file instead of controller. Because it can add some performance overheads, suppose someone is using default datagrid in that case $orders variable is worthless. 

Following the code user can add in blade file and have the orders.

<?php $orders = app('Webkul\Order\Repositories\OrderRepository')->findWhere([''customer_id' => auth()->guard('customer')->user()->id,]);

**Pull request closed without merging**